layers/+lang/typescript/packages.el

```diff
@@ -75,6 +75,7 @@
                              "gu" tide-references
                              "hh" tide-documentation-at-point
                              "rr" tide-rename-symbol
+                             "rf" tide-fix
```

Comment:
I've had this bound to just "f" for a while. "r" is "rename" at the moment. You could change that to "refactor". I don't know how @syl20bnr feels about single letter commands, but I prefer just "f" because of how frequently I use it. Also, "rf" is a single finger combination which makes it harder to type.

Reply:
Good points. I would definitely rename "r" to "refactor", I didn't notice it was "rename" at the moment. I believe this is the common practice in other layers. As for the single finger point, this is true, but I'm not sure what key would be suited better while being descriptive. Maybe "x" or "i"?

Reply:
Any reason not to use "f"?

Reply:
Sorry, forgot to mention that. IMO it's nicer to group the refactorings, while I see the point of a quick binding. But "ri" is pretty fast and fluent, been using it for a bit now.

Reply:
Sounds good to me.

Reply:
I played w/ it a little more and I'd actually be good w/ either... rf is somewhat annoying to type, but not as much as something like rv. It may be worth it to keep the most obvious mnemonic.